frontend/src/Notifications.jsx

```diff
@@ -96,13 +97,15 @@ const Notifications = (props) => {
     const interval = setInterval(() => {
       if (isNotification(notificationStore.accumulator)) {
         notificationStore.pushNotification({
-          id: classes[0].id,
+          id: uuidv4(),
+          class_id: classes[0].id,
           category: 'classes',
           value: 'Klas 1 heeft nieuwe probleem categorieën',
           index: notificationStore.notifications.length,
         });
         notificationStore.pushNotification({
-          id: classes[0].id,
+          id: uuidv4(),
+          class_id: classes[0].id,
```

Comment:
misschien dat ik dit in een vorige pr heb gemist, maar waarom ineens een snake case?